packages/hash/playwright/tests/entity-creation.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { loginUsingTempForm } from "./shared/login-using-temp-form";
+import { resetDb } from "./shared/reset-db";
+import { expect, Locator, Page, test } from "./shared/runtime";
+
+const getCellText = async (
+  canvasLocator: Locator,
+  colIndex: number,
+  rowIndex: number,
+) => {
+  const text = await canvasLocator
+    .getByTestId(`glide-cell-${colIndex}-${rowIndex}`)
+    .textContent();
+  return text;
+};
+
+const ROW_HEIGHT = 42;
+
+const clickOnValueCell = async (
+  page: Page,
+  canvasLocator: Locator,
+  colIndex: number,
+) => {
+  const canvasPos = await canvasLocator.boundingBox();
+
+  if (!canvasPos) {
+    throw new Error("canvasPos not found");
+  }
+
+  const cellY = canvasPos.y + 20 + ROW_HEIGHT * (colIndex + 1);
```

Comment:
```suggestion
  const cellY = canvasPos.y + 20 + ROW_HEIGHT * (rowIndex + 1);
```
?

Reply:
Oops, misnamed it sorry 🤦‍♂️

Reply:
Updated here https://github.com/hashintel/hash/pull/1790/commits/29548ceb30f92b07c05ef2c19cce6daf2ad59f94